.github/workflows/backend.yml

```diff
@@ -40,3 +40,77 @@ jobs:
         m2-cache-key: 'namespace-decls'
     - run: clojure -X:dev:ee:ee-dev:drivers:drivers-dev:test:namespace-checker
       name: Check ns forms
+
+  be-tests:
+    runs-on: buildjet-2vcpu-ubuntu-2004
```

Comment:
Can we run this on GHA Runner ?

Reply:
We can!

Reply:
Just a note @diogormendes: looks like the duration is longer when running without Buildjet.